external-crates/move/crates/move-compiler/src/linters/shift_overflow.rs

```diff
@@ -0,0 +1,109 @@
+//! Detect potential overflow scenarios where the number of bits being shifted exceeds the bit width of
```

Comment:
Looks like linter+tests was accidentally brought into this PR last commit. Let's remove it so we can focus on it in its own PR

Reply:
removed